nhost/nhost.go

```diff
@@ -951,7 +951,6 @@ func (config *Configuration) Init(port string) error {
 		"S3_BUCKET=nhost",
 		fmt.Sprintf("S3_ENDPOINT=%s", GetAddress(config.Services["minio"])),
 		"S3_REGION=fake-region",
-		"S3_ROOT_FOLDER=nhost",
```

Comment:
why are we removing this one?

Reply:
Due to #203. If you look at the issue the first nhost is the bucket and the second one the root folder, for some reason the old storage wasn't using this flag when using minio.